packages/sdk/src/__tests__/__fixtures__/blog-masked.prisma

```diff
@@ -0,0 +1,37 @@
+datasource db {
+  provider = "sqlite"
+  url = "***"
+}
+
+generator client {
+  provider      = "prisma-client-js"
+  binaryTargets = ["native"]
```

Comment:
```suggestion
```

Should not be needed, is there a reason why it was set?

Reply:
I have copied [`blog.prisma`](https://github.com/prisma/prisma/blob/main/packages/sdk/src/__tests__/__fixtures__/blog.prisma#L8) and simply replaced the datasource URL with the masked equivalent

Reply:
I think we could remove on both sides this part since it's not needed, might be more confusing than anything else to ha ve it around